### PR TITLE
Initial project structure

### PR DESCRIPTION
**Pull request by kuritka**
Basic project structure

Related to (#1) ([#185 k8gb](https://github.com/AbsaOSS/k8gb/issues/185))

I created basic structure with two shared packages:
 - env
 - shell

### CODE REVIEW

File: .github/workflows/inspect.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      GOLINT: on
+    name: Inspect packages
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: go1.14 linters
+        uses: AbsaOSS/golang-pipeline/go1.14/linter@master
+
+      - name: go1.14 test
+        uses: AbsaOSS/golang-pipeline/go1.14/test@master
```
Comment (ytsarev) [inline .github/workflows/inspect.yaml:18]: Shouldn't we replace `golang-pipeline` with golangci-lint from https://github.com/AbsaOSS/gopkg/pull/2/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R3 ?

**Review by ytsarev — COMMENTED** [review]
Please clarify the linting pipeline, otherwise looks good

File: .github/workflows/inspect.yaml (hunk 1 above)
Comment (kuritka) [reply .github/workflows/inspect.yaml:18]: Good point @ytsarev. In such case, should I update https://github.com/AbsaOSS/golang-pipeline/tree/master/go1.14/linter a bit ?

Comment (kuritka) [reply .github/workflows/inspect.yaml:18]: I know @somaritane has another solution candidate on the table, but temporarily we can add update pipeline.

Comment (ytsarev) [reply .github/workflows/inspect.yaml:18]: @kuritka sounds good!

Comment (kuritka) [reply .github/workflows/inspect.yaml]: @ytsarev, I made some investigations and the question is if we shouldn't go with this one: https://golangci-lint.run/usage/install/#ci-installation

At the moment, I switched to `golangci` pipeline (viz current)

Comment (somaritane) [reply .github/workflows/inspect.yaml]: > I know @somaritane has another solution candidate on the table, but temporarily we can upgrade pipeline.

Yes, the suggestion was to use https://github.com/nektos/act which allows to run GHA workflows locally, so we don't have to maintain two different pipelines, one for local execution, another for GHA.

**Review by somaritane — APPROVED** [review]
lgtm

**Review by ytsarev — APPROVED** [review]

**Pull request merged**